### PR TITLE
Solve Issue for Subscribe Box and Add Some CSS for Layout

### PR DESCRIPTION
**Pull request by mr-vijaychauhan**
# Description

First I remove from repeated `./horizontal-slim-10_7.css` from `<body> `and include inside `<head>`. First I solved Issue for subscribed box overlap your character and not looking good and added css for layout and also added responsive css for mobile friendly. I added below screenshot so you understand very well.

*Note : Automatically format of index.html files because I am using VS CODE.

Fixes # (issue)

## Solve Issue for Subscribe Box to overlap to your character image.

# How Has This Been Tested? 

I Upload below Screenshot of Before and After of index.html layout to Easily understanad what I did.

- [ ] Before
Before__Layout

- [ ] After
After__Layout
PC_for Subscribe Modal
Responsive_Layout
Responsive__Layout

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Thank you @mr-vijaychauhan 👍 , looks like great improvements 🤓 

Please can you do the formatting of the code in its own separate PR otherwise it is impossible to see the actual code changes.

**Comment by mr-vijaychauhan** [comment]
> Thank you @mr-vijaychauhan 👍 , looks like great improvements 🤓
> 
> Please can you do the formatting of the code in its own separate PR otherwise it is impossible to see the actual code changes.

Thank You Sir to Replied. I did Changes as You Says. Please Check this Changes.

**Comment by mr-vijaychauhan** [comment]
Final Changes Done

File: index.html [hunk 1]
```diff
@@ -23,6 +23,57 @@
...
+      width: 100% !important;
+      margin-bottom: 0 !important;
+      -webkit-box-shadow: none !important;
+      -moz-box-shadow: none !important;
+      box-shadow: none !important;
+      border: none !important;
+    }
+
+    /* Responsive layout */
+    @media (max-width: 800px) {
```
Comment (eddiejaoude) [inline index.html:65]: I think it might be best to use the bootstrap responsive classes?

**Comment by eddiejaoude** [comment]
Thank you 👍 . 

The last word on a new line by itself looks a bit odd, can it be made wider or smaller?

Screenshot 2020-10-25 at 22 14 59

The form breaks outside the card

Screenshot 2020-10-25 at 22 15 10

**Comment by mr-vijaychauhan** [comment]
Hi Sir, 

# Description

I added bootstrap class for Subscribe box as well as Responsive Devices. I Fixed Problem Form breaks outside the card. I added Bootstrap Class For Your Content Justify. I tried to fix your content overlapping your character images.

- [ ] Before (Without Justify Content)
Screenshot (52)

- [ ] After (With Justify Content and Fix Problem to Overlapping Character Image)
Screenshot (53)

- [ ] I cann't fix this Overlapping Problem in Mobile Device because Your Character Inside Your Body
Screenshot (54)

I tried My best to Solve Your Problem.

Thanks  !

**Comment by eddiejaoude** [comment]
This looks much better now 👍 . Thank you 🤓

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍

**Pull request merged**